cmd/dice on develop branch doesn't build

`cmd/dice` on default branch `develop` doesn't build.

Output:

```console
$ go get -u github.com/travis-g/dice/...
# github.com/travis-g/dice/cmd/dice
go/src/github.com/travis-g/dice/cmd/dice/main.go:24:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/travis-g/dice/cmd/dice/main.go:28:4: unknown field 'EnvVar' in struct literal of type cli.StringFlag
go/src/github.com/travis-g/dice/cmd/dice/main.go:30:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/travis-g/dice/cmd/dice/main.go:34:4: unknown field 'EnvVar' in struct literal of type cli.StringFlag
go/src/github.com/travis-g/dice/cmd/dice/main.go:39:17: cannot use cli.StringFlag literal (type cli.StringFlag) as type cli.Flag in slice literal:
	cli.StringFlag does not implement cli.Flag (Apply method has pointer receiver)
go/src/github.com/travis-g/dice/cmd/dice/main.go:43:4: unknown field 'EnvVar' in struct literal of type cli.StringFlag
go/src/github.com/travis-g/dice/cmd/dice/main.go:47:15: cannot use []cli.Command literal (type []cli.Command) as type []*cli.Command in assignment
```

--- a/cmd/dice/main.go
+++ b/cmd/dice/main.go
@@ -21,30 +21,30 @@ func main() {
 	// globalFlags should be set up so that they can be used anywhere in the
 	// command
 	globalFlags := []cli.Flag{
-		cli.StringFlag{
-			Name:   "format",
-			Value:  "",
-			Usage:  "output format",
-			EnvVar: "FORMAT",
+		&cli.StringFlag{
+			Name:    "format",
+			Value:   "",
+			Usage:   "output format",
+			EnvVars: []string{"FORMAT"},
 		},
-		cli.StringFlag{
-			Name:   "field",
-			Value:  "",
-			Usage:  "output specific field (unimplemented)",
-			EnvVar: "FIELD",
+		&cli.StringFlag{
+			Name:    "field",
+			Value:   "",
+			Usage:   "output specific field (unimplemented)",
+			EnvVars: []string{"FIELD"},
 		},
 	}
 
 	httpFlags := []cli.Flag{
-		cli.StringFlag{
-			Name:   "http",
-			Value:  ":6436", // base64("d6")
-			Usage:  "HTTP service address",
-			EnvVar: "HTTP",
+		&cli.StringFlag{
+			Name:    "http",
+			Value:   ":6436", // base64("d6")
+			Usage:   "HTTP service address",
+			EnvVars: []string{"HTTP"},
 		},
 	}
 
-	cmd.Commands = []cli.Command{
+	cmd.Commands = []*cli.Command{
 		{
 			Name:    "eval",
 			Aliases: []string{"e"},

